src/gluonts/gp/gaussian_process.py

```diff
@@ -250,7 +250,7 @@ def sample(self, mean: Tensor, covariance: Tensor) -> Tensor:
             self._compute_cholesky_gp(
                 covariance, self.prediction_length, self.sample_noise
             ),
-            float_type=self.float_type,
+            dtype=self.float_type,
```

Comment:
`self.dtype`?

Reply:
As mentioned, this would trigger changes on the `gp_forecaster` as well which I would like to do separately. The `gp_forecaster` also needs to be updated to use proper `dtype`. Simply passing `dtype` to `GluonEstimator` does not suffice.

Reply:
Yes, I think the `gp_forecaster` will work will either `dtype`.  We may just need to do some renaming of the variables.